rpc/api.proto

```diff
@@ -1100,6 +1100,8 @@ message ValidateAddressResponse {
 		StakeSubChangeTy = 9;
 		PubkeyAltTy = 10;
 		PubkeyHashAltTy = 11;
+		TreasuryAddTy = 12;
+		TreasuryGenTy = 13;
```

Comment:
These were added but not used. Is this correct?

Reply:
I believe that if you verified a script has of a treasury add or gen that your wallet has the script to, this would be returned. On master, testing for treasury is off, but this pr also turns it on. I will try it out and see if I get the return.

Reply:
yeah, this looks right.  these constants correlate to those of txscript.ScriptClass and must be kept in sync.  There's a type conversion between the txscirpt.ScriptClass to this protobuf enum so the values must be the same.